sdk/src/utils/resolvers.ts

```diff
@@ -0,0 +1,52 @@
+import { Resolvers, MeshContext } from "../../.graphclient";
+
+export const resolvers: Resolvers = {
+  Claim: {
+    chainName: (root, args, context, info) => context.chainName || "hypercerts-testnet", // The value we provide in the config
+  },
+  ClaimToken: {
+    chainName: (root, args, context, info) => context.chainName || "hypercerts-testnet", // The value we provide in the config
```

Comment:
wow. Amazing magic. I can't imagine how annoying it was to figure this out....

Reply:
If it works 🙏 

I'm not seeing errors on my end so it appears to work, but there's not a gallery view anymore right?